### ISSUE TITLE
Sync problems - loss of note when edited on another device

### ISSUE THREAD

**Issue by robme** [turtl/tracker#376]
After spending a lot of time editing a note on the Android app, I have then opened that note on the Windows program, and realised that I've lost it all. Opening it again on Android will then show whatever the Windows one has, and everything I wrote in Android is gone. I assume it did not sync before I started editing on Windows?

I think Turtl needs some kind of conflicting changes detection when syncing.

**Comment by lambdagolem** [comment]
I think this is related/the same as  #317

**Comment by robme** [comment]
Yes it sounds like the same problem. The current behaviour seems to be the latest device overwrites it.

In Evernote when this occurs, it saves the conflicting changes in another note so that they are not lost. I appreciate that might be tricky to implement. I'm currently just being extra careful and making sure my phone has synced before opening the desktop program.
